test.util/src/main/java/org/kaazing/test/util/MethodExecutionTrace.java

```diff
@@ -32,49 +36,55 @@
     @Rule
     public MethodRule testExecutionTrace = new MethodExecutionTrace();
  */
-public class MethodExecutionTrace extends TestWatchman {
-    //if we ever want the full test name (including method),
-    //we can call the public static method which retrieves this
-    //variable. This variable is set in the starting method, and
-    //and unset in the fail/success methods
-    private static String currentFullTestName;
-
-    public static String getFullTestName() {
-        return currentFullTestName;
-    }
+public class MethodExecutionTrace extends TestWatcher {
 
     /**
      * Use this constructor to set up a particular log4j configuration using a properties file,
-     * e.g. to use MemoryAppender.
+     * or null if you do not want to load any log4j configuration properties.
      * See gateway.server src/test/resources/log4j-trace.properties for an example.
      * @param log4jConfigPropertiesFileName
      */
-    public MethodExecutionTrace(String log4jConfigPropertiesFileName) {
+    public MethodExecutionTrace(String log4jPropertiesResourceName) {
         MemoryAppender.initialize();
```

Comment:
Good idea but I don't have time to bite that off right now. Adding a TODO note in the class.